.circleci/config.yml

```diff
@@ -35,11 +35,11 @@ jobs:
 
       - restore_cache:
           keys:
-            - v12-model_cache-{{ checksum "setup.py" }}
+            - v13-model_cache-{{ checksum "setup.py" }}
```

Comment:
We should do another checksum (maybe of some other file) instead of relying on increasing the version number.